plugin-api/src/main/java/com/gentics/mesh/plugin/env/PluginEnvironment.java

```diff
@@ -25,12 +26,19 @@
 	Vertx vertx();
 
 	/**
-	 * Create a new admin client.
+	 * Create a new admin client with api version 1.
 	 * 
 	 * @return
 	 */
 	MeshRestClient createAdminClient();
 
+	/**
+	 * Creates a new admin client from the specified version.
+	 * 
+	 * @return
```

Comment:
Add param - add return doc. This is a public API method and thus should have full javadoc